app/lib/user-groups.js

```diff
@@ -0,0 +1,21 @@
+const GROUP_META = require('../data/groups.json');
+
+const getUserGroupLandingRoute = groups => {
+  let priorityGroup = GROUP_META[groups[0]];
+
+  for (let x = 1; x < groups.length; x++) {
+    const curr = GROUP_META[groups[x]];
```

Comment:
This may be undefined as "Realm Administrator" may not be the first group in the list

Reply:
I assumed that Realm Administrator does not have access to the portal, but I will add it into group list

Reply:
I think this function needs to be able to handle and ignore unknown groups.

Reply:
definitely, it shouldn't crash. I am thinking to set unkown groups to just guest